db/versions/0091.yml

```diff
@@ -0,0 +1,391 @@
+version: 91
+description: migrate existing azure queue messages into new tables
+
+migrationScript: 0091-migration.sql
+downgradeScript: 0091-downgrade.sql
+
+methods:
+  # those methods and the azure messages table would be dropped in next release
+  azure_queue_get:
+    deprecated: true
+  azure_queue_count:
+    deprecated: true
+  azure_queue_delete:
+    deprecated: true
+  azure_queue_update:
+    deprecated: true
+  azure_queue_put_extra:
+    deprecated: true
+  azure_queue_delete_expired:
+    deprecated: true
+
+  queue_pending_tasks_get:
```

Comment:
there are 4 `queue_*_get` methods that are doing the same that `azure_queue_get` was doing - selecting rows with `FOR UPDATE SKIP LOCKED` to guarantee that rows would only be fetched once.